shell/platform/android/test_runner/build.gradle

```diff
@@ -47,6 +47,7 @@ android {
     testImplementation "com.google.android.play:core:1.8.0"
     testImplementation "com.ibm.icu:icu4j:69.1"
     testImplementation "org.mockito:mockito-core:3.11.2"
+    testImplementation "org.mockito:mockito-inline:3.11.2"
```

Comment:
is this needed? When I remove it, I don't see any of OutOfMemory errors, and others.

Reply:
@blasten I need to mock [WindowInsetsControllerCompat](https://developer.android.com/reference/androidx/core/view/WindowInsetsControllerCompat) which is a final class, and this seems to be the solution to be able to do so.

Reply:
https://github.com/flutter/engine/pull/30257 should fix the issues you are seeing